Resources/Prototypes/Entities/Clothing/Head/misc.yml

```diff
@@ -97,6 +97,17 @@
   - type: Clothing
     sprite: Clothing/Head/Misc/head_mirror.rsi
 
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatFakePoppy
+  name: fake poppy
+  description: A fake poppy. Can be worn on the head.
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Misc/fakepoppy.rsi
```

Comment:
You don't need to duplicate the poppy sprite to make it usable as clothing. 

1. Remove the fakepoppy.rsi sprites and folder.
2. Set the sprite path to that of the normal poppy sprite folder: https://github.com/space-wizards/space-station-14/tree/bb2981400cb34d476dd7f6f90aba4d0209dab047/Resources/Textures/Objects/Specific/Hydroponics/poppy.rsi
3. Add `state: produce` as a new line under the sprite component. This makes it use the `produce.png` sprite as the icon, which should be what you want.

You wouldn't need to add a state for the Clothing component though since that defaults to the file named `equipped-HELMET.png`